sources/web/datalab/reverseProxy.ts

```diff
@@ -48,12 +48,22 @@ export function getRequestPort(request: http.ServerRequest, path: string): strin
 }
 
 /**
+ * Checks if a request should be exempted from reverse proxying to the gateway
+ */
+function shouldSkipGateway(port: String) {
```

Comment:
It took me a long time to realize why this logic was doing what it was doing, and I think that a slightly different structure would have made it easier to understand the reasoning.

I'd suggest the following:

1. Change the name of this function to something like 'isLocalPort'
2. Reorder the conditional branches in the 'handleRequest' method. With this change, what was previously the `else` branch is conceptually the default branch, so it feels like it should come first in the order.

Alternatively, we could make the 'isLocalPort' method check the 'KG_URL' environment variable.